src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        if ((email.equals("bob@i.ua") || email.equals("alice@i.ua")) && password.equals("1234")) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
you should create an instance of UserService here and use implemented findByEmail to implement login method